packages/tools/guiEditor/src/components/sceneExplorer/treeItemSelectableComponent.tsx

```diff
@@ -140,16 +148,10 @@ export class TreeItemSelectableComponent extends React.Component<ITreeItemSelect
         };
         const entity = this.props.entity;
 
-        if (!entity.reservedDataStore) {
-            entity.reservedDataStore = {
-                isExpanded: true,
-                setExpandedState: (expanded: boolean) => (entity.reservedDataStore.isExpanded = expanded),
-            };
-        }
-        const isExpanded = entity.reservedDataStore.isExpanded || Tools.LookForItems(this.props.entity, this.props.selectedEntities);
-        entity.reservedDataStore.isExpanded = isExpanded;
+        const isExpanded = this.state.isExpanded || Tools.LookForItems(this.props.entity, this.props.selectedEntities);

```

Comment:
I wonder what is the need for the code `Tools.LookForItems(this.props.entity, this.props.selectedEntities)`. I saw that it is part of the old code as well, but it seems like `isExpanded`  was used later in the code as well, but you are using `this.state.isExpanded`, which might make this definition redundant?

Reply:
Thanks for the catch! I meant to use isExpanded to keep it close to the original code, but changed it to this.state.isExpanded while debugging and forgot to change it back.